src/bootstrap/build/sanity.rs

```diff
@@ -79,7 +79,7 @@ pub fn check(build: &mut Build) {
         }
 
         // Make sure musl-root is valid if specified
-        if target.contains("musl") {
+        if target.contains("musl") && target.contains("x86_64") {
```

Comment:
Right, we'd want to use static linking if it worked, but I don't think it would even get past compiling libstd today.